src/panel/category/CategoryPanel.jsx

```diff
@@ -134,10 +137,28 @@ const CategoryPanel = ({ poiFilters = {}, bbox }) => {
     </>;
   }
 
+  const NavHeader = () => {
+    if (isMobile) {return null;}
+
+    if (shouldShowBackToQwant()) {
+      return (
+        <PanelNav>
+          <BackToQwantButton />
+        </PanelNav>
+      );
+    }
+
+    return null;
+  };
+
   return <Panel
     resizable
+    renderHeader={<NavHeader isMobile={isMobile} />}
     minimizedTitle={_('Unfold to show the results', 'categories')}
     className={classnames('category__panel', { 'panel--pj': dataSource === sources.pagesjaunes })}
+    floatingItemsLeft={[
+      isMobile && <BackToQwantButton key="back-to-qwant" />,
```

Comment:
We have to use `shouldShowBackToQwant` here also, otherwise it's always present on mobile.